dotcom-rendering/src/web/components/ArticleMeta.stories.tsx

```diff
@@ -124,6 +127,9 @@ export const BrandingStory = () => {
 				tags={tagsWithBylineImage}
 				primaryDateline="Sun 12 Jan 2020 18.00 GMT"
 				secondaryDateline="Sun 12 Jan 2020 21.00 GMT"
+				isCommentable={false}
+				discussionApiUrl=""
+				shortUrlId=""
```

Comment:
General style question - when do we use `empty string` vs `undefined` or other none-able values?

Reply:
I don't have strong opinions on this personally but I think typescript might. The type of `discussionApiUrl` is `string` and it is required so by passing `""` here we're kind of keeping TS happy. You could argue this is cheating but the api call is mocked in this case so maybe it's better not passing it so it's very clear it isn't being used?

Reply:
Ah, OK, so it is an expected string outside of testing? In which case this makes perfect sense.